### PR TITLE
Adjust numerical columns to the right

### LINKED ISSUE #273: right justify columns with numbers
E.g. on the atlas page

Screen Shot 2021-04-23 at 4 51 30 PM

### PR DESCRIPTION
**Pull request by onursumer**
Fix #273

### CODE REVIEW

File: lib/dataSchemaHelpers.ts [hunk 1]
```diff
@@ -105,8 +105,67 @@ export enum SchemaDataId {
...
+    [SchemaDataId.PercentInflamInfiltration]: true,
+    [SchemaDataId.PercentLymphocyteInfiltration]: true,
+    [SchemaDataId.PercentMonocyteInfiltration]: true,
+    [SchemaDataId.PercentNecrosis]: true,
+    [SchemaDataId.PercentNeutrophilInfiltration]: true,
+    [SchemaDataId.PercentNormalCells]: true,
+    [SchemaDataId.PercentStromalCells]: true,
+    [SchemaDataId.PercentTumorCells]: true,
+    [SchemaDataId.PercentTumorNuclei]: true,
+};
```
Comment (onursumer) [inline lib/dataSchemaHelpers.ts:167]: We wouldn't need any of these if `DataSchemaData` has better values for the `type` field. Unfortunately it is almost always `rdfs:Class` which is not very helpful.

Comment (inodb) [reply lib/dataSchemaHelpers.ts:167]: hmm yeah I think it would be good to fix the schema. @milen-sage is this something that would be easy to add?

**Comment by inodb** [comment]
Thanks for fixing this! Noticed the alignment looks slightly funky on the atlas tab of the explore page:

Screen Shot 2021-04-29 at 8 18 03 AM

Might be resolved by right adjusting the headers too? Maybe viewers can be slightly more wide and Cases and Biospecimen more slim?

**Comment by onursumer** [comment]
> Might be resolved by right adjusting the headers too? Maybe "Viewers" column can be slightly more wide and Cases and Biospecimen more slim?

@inodb Aligning only some of the column headers seems tricky (see https://github.com/jbetancur/react-data-table-component/issues/788). I will try to find a workaround for this.

**Comment by onursumer** [comment]
@inodb found a workaround to align column headers only for Atlas table. Let me know if this is good to go.

**Comment by inodb** [comment]
Thanks!!

**Pull request merged**